### PR TITLE
Update nl/lang.php

### PR DESCRIPTION
**Pull request by thomasilmer**
Added new Dutch translations and translated untranslated messages.

### CODE REVIEW

File: lang/nl/lang.php [hunk 1]
```diff
@@ -30,16 +31,21 @@
...
         'clear_cache_link' => 'Leeg cache',
         'clear_cache_loading' => 'Applicatie cache legen...',
         'clear_cache_success' => 'De applicatie cache is succesvol geleegd.',
         'clear_cache_hint' => 'Het is verstandig om regelmatig op <strong>Leeg cache</strong> te klikken om de veranderingen te zien op de website.',
         'scan_messages_link' => 'Zoek naar nieuwe berichten',
         'scan_messages_loading' => 'Zoeken naar nieuwe berichten...',
         'scan_messages_success' => 'De thema bestanden zijn succesvol gescand!',
-        'scan_messages_hint' => 'Klikken op <strong>Zoeken naar nieuwe berichten</strong> controleert de actieve thema bestanden voor nieuwe berichten om te vertalen. ',
+        'scan_messages_hint' => 'Klikken op <strong>Zoeken naar nieuwe berichten</strong> controleert de actieve thema bestanden voor nieuwe berichten om te vertalen.',
+	'scan_messages_process' => 'Dit proces zal het actieve thema scannen voor berichten die kunnen worden vertaald.',
```
Comment (LukeTowers) [inline lang/nl/lang.php]: Missing a space here

File: lang/nl/lang.php [hunk 2]
```diff
@@ -30,16 +31,21 @@
     ],
     'messages' => [
         'title' => 'Vertaal berichten',
-		'description' => 'Wijzig berichten',
+	'description' => 'Wijzig berichten',
```
Comment (LukeTowers) [inline lang/nl/lang.php]: Missing a space here

**Comment by thomasilmer** [comment]
I have added the spaces now.

**Comment by LukeTowers** [comment]
I don't see them in the Files Changed tab

**Comment by LukeTowers** [comment]
Still don't see them @thomasilmer

**Pull request closed without merging**